ui/dashboardApp/layout/main/index.tsx

```diff
@@ -84,27 +43,9 @@ export default function App({ registry }) {
                 collapsedWidth={siderCollapsedWidth}
                 animationDelay={0}
               />
-              <animated.div
-                className={styles.contentBack}
-                style={{
-                  left: `${siderWidth}px`,
-                  transform: transContentBack.x.interpolate(
-                    (x) => `translate3d(${x}px, 0, 0)`
-                  ),
-                }}
-              ></animated.div>
             </>
           )}
-          <div
-            className={styles.content}
-            style={
-              appOptions.hideNav
```

Comment:
will this break the `hideNav` functionality?

Reply:
won't.